pkg/agent/baker_test.go

```diff
@@ -1317,7 +1324,14 @@ var _ = Describe("Assert generated customData and cseCmd for Windows", func() {
 		Entry("AKSWindows2019 with k8s version 1.19 + FIPS", "AKSWindows2019+K8S119+FIPS", "1.19.0",
 			func(config *datamodel.NodeBootstrappingConfiguration) {
 				config.FIPSEnabled = true
-			}))
+			}),
+		Entry("AKSWindows2019 with SecurityProfile", "AKSWindows2019+SecurityProfile", "1.26.0",
```

Comment:
Prefer to use AKSWindows2022 since the default Windows OS is 2022 in k8s 1.25+

Reply:
Currently all cases are on windows2019, and it's hardcoded:

Distro:              datamodel.Distro("aks-windows-2019"),

WindowsSku:                    "aks-2019-datacenter-core-smalldisk-2011",


Do you suggest to override the SKU in this case, or update for all cases later?

Reply:
I see. You can still use Windows2019. We will update it later.